example/src/View.elm

```diff
@@ -32,6 +33,23 @@ formView form =
 
         superpowerOptions =
             List.map (\i -> ( i, String.toUpper i )) superpowers
+
+        disableSubmit =
+            Set.isEmpty <| Form.getChangedFields form
+
+        submitBtnAttributes =
+            [ onClick Form.Submit
+            , class <|
```

Comment:
I'm not sure what you mean by this? Should I just move class composition up to extra variable classList? Because use List of Strings here looks still not necessary here.